sacred/utils.py

```diff
@@ -315,6 +316,60 @@ def __init__(
         super().__init__(message, print_traceback, filter_traceback, print_usage)
 
 
+class FilteredTracebackException(tb.TracebackException):
```

Comment:
Can you add a docstring explaining why we need this class?

Reply:
Done